Mechanical/Differential Pivot/README.md

```diff
@@ -1,38 +1,147 @@
 # Differential Pivot
 The Differential Pivot helps to offload forces from one side of the rover to the other while climbing. The differential pivot is crucial in the rocker-bogie suspension system and is designed to help keep all 6 wheels on the ground at all times. It also allows a second attachment point for the body so that it does not free rotate about the center bar that runs through it and connects the two sides of the rocker-bogie.
 
-![Differential Pivot](Latex%20Docs/Pictures/Differential%20Pivot.PNG)
+![Differential Pivot](images/differential_pivot.PNG) 
 
 ## Features
   * Uses inexpensive RC turnbuckles to give multi-degree rotation about 2 separate points, allowing the geometry to be greatly simplified
 
-## Information about module
+## Mechanical Interface/Attachments to Rover:
+  * 1/2 inch clamping hub to 1/2 inch aluminum rod on the body
+  * 1/2 inch aluminum rod to 1/2 inch clamping hub on rocker-bogie arms
 
-| Author/Creator            | JPL       |
-| --------------            | -------            |
-| Cost without 3D printer   | 84.90              |
-| Cost with 3D printer      | 84.90              |
-| Date Added                | 4/11               |
-| Group/Organization/School | JPL                |
-| Time to build             | 2 hrs              |
+## Machining/Fabrication
 
-## Skills
-  * Band Saw/Dremel
-  * Drill Press/Hand Drill
+1. Aluminum Rods: Cutting
 
-## Tools
+    Parts/Tools Necessary
 
-### Mandatory 
-  * Band saw or Dremel
-  * Drill press or Hand Drill
-  * Allen Key set
-  * Imperial Wrench Set
+    | Item | Ref | Qty | Image |
+    | :--- | :-- | :-- | :---: |
+    | 0.5" x 3' Aluminum Tube | S16 | 1 | ![S16](/images/components/Structural/S16.png) |
+    | 0.5" x 4" Aluminum Tube | S18 | 4 | ![S18](/images/components/Structural/S18.png) |
+    | Metal Hacksaw or Bandsaw | D4 | 1  | ![D4](/images/components/Tools/D4.png) |
+    | Vice clamp or C clamps | D5 | 1 | ![D5](/images/components/Tools/D5.png) |
 
-### Optional
+    1. Take the 3 foot piece of 0.5 inch aluminum rod S16 and cut it into one 15 inch piece and one 13.5 inch piece. These will now be referred to as the parts S16A and S16B respectively. In addition, take two of the 4 inch aluminum rods S18 and cut them down to 3 inches in length as well.
```

Comment:
**bold** the S#'s in the text?

(again, let me know if you want me to fix, I can do this super quickly in sublime text)